src/network/tc/etf.h

```diff
@@ -0,0 +1,37 @@
+/* SPDX-License-Identifier: LGPL-2.1+
+ * Copyright © 2020 Intel Corporation */
+#pragma once
+
+#include "conf-parser.h"
+#include "qdisc.h"
+#include "time-util.h"
+
+#define CLOCKID_INVALID (-1)
+
+static const struct static_clockid {
+      const char *name;
+      clockid_t clockid;
+} clockids_sysv[] = {
+      { "REALTIME", CLOCK_REALTIME },
+      { "TAI", CLOCK_TAI },
+      { "BOOTTIME", CLOCK_BOOTTIME },
+      { "MONOTONIC", CLOCK_MONOTONIC },
+      { NULL }
+};
+
+typedef struct EarliestTxTimeFirst {
+        QDisc meta;
+
+        __s32 clockid;
+        __s32 delta;
```

Comment:
`__s32` is a kernel thing. Please use `clockid_t` for `clockid`, and `int32_t` for `delta`.